utils/dc-chain/scripts/newlib.mk

```diff
@@ -43,21 +44,19 @@ fixup-sh4-newlib-apply:
 # to define _POSIX_THREADS
 # pthreads to kthreads mapping
 # so KOS includes are available as kos/file.h
-# kos/thread.h requires arch/arch.h
-# arch/arch.h requires dc/video.h
+# kos/thread.h requires arch/irq.h
 	cp $(kos_base)/include/pthread.h $(newlib_inc)
 	cp $(kos_base)/include/sys/_pthread.h $(newlib_inc)/sys
 	cp $(kos_base)/include/sys/sched.h $(newlib_inc)/sys
+	cp $(kos_base)/include/sys/lock.h $(newlib_inc)/sys
 ifndef MINGW32
 	ln -nsf $(kos_base)/include/kos $(newlib_inc)
 	ln -nsf $(kos_base)/kernel/arch/dreamcast/include/arch $(newlib_inc)
-	ln -nsf $(kos_base)/kernel/arch/dreamcast/include/dc   $(newlib_inc)
```

Comment:
The dc folder is just no longer needed as kos/thread no longer chains into needed anything from there.